{
  "cve": "CVE-2023-51656",
  "mitre": {
    "cpes": [],
    "created": "2023-12-21T11:47:57.912000+00:00",
    "description": "Deserialization of Untrusted Data vulnerability in Apache IoTDB.This issue affects Apache IoTDB: from 0.13.0 through 0.13.4.\n\nUsers are recommended to upgrade to version 1.2.2, which fixes the issue.\n\n",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {},
      "cvssV4_0": {}
    },
    "mitre_repo_path": "cves/2023/51xxx/CVE-2023-51656.json",
    "references": [
      "http://www.openwall.com/lists/oss-security/2023/12/21/5",
      "https://lists.apache.org/thread/zy3klwpv11vl5n65josbfo2fyzxg3dxc"
    ],
    "title": "Apache IoTDB: Unsafe deserialize map in Sync Tool",
    "updated": "2024-08-02T22:40:34.137000+00:00",
    "vendors": [],
    "weaknesses": [
      "CWE-502"
    ]
  },
  "nvd": {
    "cpes": [
      "cpe:2.3:a:apache:iotdb:*:*:*:*:*:*:*:*"
    ],
    "created": "2023-12-21T12:15:08.050000+00:00",
    "description": "Deserialization of Untrusted Data vulnerability in Apache IoTDB.This issue affects Apache IoTDB: from 0.13.0 through 0.13.4.\n\nUsers are recommended to upgrade to version 1.2.2, which fixes the issue.\n\n",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {
        "score": 9.8,
        "vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
      },
      "cvssV4_0": {}
    },
    "nvd_repo_path": "2023/CVE-2023-51656.json",
    "references": [
      "http://www.openwall.com/lists/oss-security/2023/12/21/5",
      "https://lists.apache.org/thread/zy3klwpv11vl5n65josbfo2fyzxg3dxc"
    ],
    "title": null,
    "updated": "2024-01-02T17:59:52.730000+00:00",
    "vendors": [
      "apache",
      "apache$PRODUCT$iotdb"
    ],
    "weaknesses": [
      "CWE-502"
    ]
  },
  "opencve": {
    "changes": [],
    "cpes": {
      "data": [
        "cpe:2.3:a:apache:iotdb:*:*:*:*:*:*:*:*"
      ],
      "providers": [
        "nvd"
      ]
    },
    "created": {
      "data": "2023-12-21T11:47:57.912000+00:00",
      "provider": "mitre"
    },
    "description": {
      "data": "Deserialization of Untrusted Data vulnerability in Apache IoTDB.This issue affects Apache IoTDB: from 0.13.0 through 0.13.4.\n\nUsers are recommended to upgrade to version 1.2.2, which fixes the issue.\n\n",
      "provider": "mitre"
    },
    "metrics": {
      "cvssV2_0": {
        "data": {},
        "provider": null
      },
      "cvssV3_0": {
        "data": {},
        "provider": null
      },
      "cvssV3_1": {
        "data": {
          "score": 9.8,
          "vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
        },
        "provider": "nvd"
      },
      "cvssV4_0": {
        "data": {},
        "provider": null
      },
      "kev": {
        "data": {},
        "provider": null
      },
      "ssvc": {
        "data": {},
        "provider": null
      },
      "threat_severity": {
        "data": null,
        "provider": null
      }
    },
    "references": {
      "data": [
        "http://www.openwall.com/lists/oss-security/2023/12/21/5",
        "https://lists.apache.org/thread/zy3klwpv11vl5n65josbfo2fyzxg3dxc"
      ],
      "providers": [
        "mitre",
        "nvd"
      ]
    },
    "title": {
      "data": "Apache IoTDB: Unsafe deserialize map in Sync Tool",
      "provider": "mitre"
    },
    "updated": {
      "data": "2024-08-02T22:40:34.137000+00:00",
      "provider": "mitre"
    },
    "vendors": {
      "data": [
        "apache",
        "apache$PRODUCT$iotdb"
      ],
      "providers": [
        "nvd"
      ]
    },
    "weaknesses": {
      "data": [
        "CWE-502"
      ],
      "providers": [
        "mitre",
        "nvd"
      ]
    }
  }
}